packages/sdk/src/actor/behaviors/penBehavior.ts

```diff
@@ -0,0 +1,28 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { ToolBehavior } from '.';
+import { TransformLike } from '../..';
+
+
+interface DrawData {
+	transform: TransformLike;
+	// Potentially additional data to come, such as:
+	// pressure: number;
+}
+
+interface PenEventData {
+    drawData: DrawData[];
+}
+
+/**
+ * Pen behavior class containing the target behavior actions.
+ */
+export abstract class PenBehavior extends ToolBehavior<PenEventData> {
```

Comment:
Afterthought: Mark ToolBehavior and PenBehavior @hidden so they don't show up in generated documentation (yet).

Reply:
Good call.  Missed that.  Will add in a new PR.